pinlockview/src/main/java/com/andrognito/pinlockview/PinLockView.java

```diff
@@ -254,6 +278,207 @@ public void setTextSize(int textSize) {
+    }
+
+    /**
+     * Set the color fo the delete button
+     *
+     * @param buttonColor the color of the delete button
+     */
+    public void setDeleteButtonColor(int buttonColor) {
+        this.mDeleteButtonColor = buttonColor;
+        mCustomizationOptionsBundle.setDeleteButtonColor(buttonColor);
+        mAdapter.notifyDataSetChanged();
+    }
+
+    /**
+     * Checks if the numbers text is bold
+     *
+     * @return returns true if numbers are bolded, false otherwise
+     */
+    public boolean isNumbersTextBold() {
+        return mNumbersTextBold;
+    }
+
+    /**
+     * Dynamically set the boldness of numbers text
+     *
+     * @param isNumbersBold true if the numbers text should be bold, false otherwise
+     */
+    public void setNumbersTextBold(boolean isNumbersBold) {
+        this.mNumbersTextBold = isNumbersBold;
+        mCustomizationOptionsBundle.setIsNumbersTextBold(isNumbersBold);
+        mAdapter.notifyDataSetChanged();
+    }
+
+    /**
+     * Checks if the letters text is bold
+     *
+     * @return returns true if letters are bolded, false otherwise
+     */
+    public boolean isLettersTextBold() {
+        return mLettersTextBold;
+    }
+
+    /**
+     * Dynamically set the boldness of letters text
+     *
+     * @param isLettersBold true if the letters text should be bold, false otherwise
+     */
+    public void setLettersTextBold(boolean isLettersBold) {
+        this.mLettersTextBold = isLettersBold;
+        mCustomizationOptionsBundle.setIsLettersTextBold(isLettersBold);
+        mAdapter.notifyDataSetChanged();
+    }
+
+    /**
+     * Are the letters in the keypad buttons shown
+     *
+     * @return returns true if shown, false otherwise
+     */
+    public boolean isShownLetters() {
+        return mShowLetters;
+    }
+
+    /**
+     * Dynamically set if the letters in buttons should be shown
+     *
+     * @param showLetters true if the letters in buttons should be shown, false otherwise
+     */
+    public void setShownLetters(boolean showLetters) {
+        this.mShowLetters = showLetters;
+        mCustomizationOptionsBundle.setShowLetters(showLetters);
+        mAdapter.notifyDataSetChanged();
+    }
+
+
+    /**
+     * Is the delete button of default color (white)
+     *
+     * @return returns true if it is default color, false otherwise
+     */
+    public boolean isDefaultDeleteColor() {
+        return mDeleteButtonDefault;
+    }
+
+    /**
+     * Dynamically set if the delete button is of default color
+     *
+     * @param showLetters true if the letters in buttons should be shown, false otherwise
+     */
+    public void setDefaultDeleteColor(boolean isDefaultColor) {
+        this.mDeleteButtonDefault = isDefaultColor;
+        mCustomizationOptionsBundle.setDeleteButtonDefault(isDefaultColor);
+        mAdapter.notifyDataSetChanged();
+    }
+
+    /**
+     * Are we using deprecated color options for color of numbers and letters
```

Comment:
Yes. In the original library,

- _keypadTextColor_ and _keypadTextSize_ affected the font of the numbers (I made it affect the letters too)

But I added:

_keypadLettersTextSize, keypadLettersTextColor_ and the equivalent for colors so that the sizes and colors of the letters and numbers can be set independently.

I wanted to account for the case where a user may accidentally use all of these properties- _keypadUseDeprecatedColorOptions_ would make it explicit which one the user would want to use.